mantidimaging/gui/ui/main_window.ui

```diff
@@ -193,6 +194,11 @@
     <string>Load projection angles...</string>
    </property>
   </action>
+  <action name="actionLoadImages">
+   <property name="text">
+    <string>Load images stack</string>
```

Comment:
```suggestion
    <string>Load images</string>
```
I don't like "Load images stack" but I'm also not sure this is much better

Reply:
The reason I added "stack" was simply because I wanted to convey that it was a singular set of images we could go with "Load image set", with the myriad of options we now have the File menu I am starting to wonder if clarity is going to become an issue from a random User's perspective.